CHANGES.md

```diff
@@ -24,6 +24,94 @@ this file to understand what changed.
+    - add e2e UI tests for responsibleaidashboard-housing-classification-model-debugging notebook
+    - fix e2e UI What-if sort test
+    - add e2e UI tests for Feature Importance - responsibleaidashboard-diabetes-decision-making notebook
+    - add e2e UI tests for What-if, Model Overview and Data explorer - responsibleaidashboard-diabetes-decision-making notebook
+    - add e2e UI tests for Aggregate Causal Analysis - diabetes-decision-making notebook
+    - add multiclass classification dataset to UI test app and set up basic model-assessment tests for Model Overview
+    - add notebook test for pytorch example notebook and add pytorch to CI
+    - move to new fluentUI dependency
+    - correctly annotate python warnings
+    - separate out raiwidgets and UI tests into its own workflow
+    - add js and raiwidget code coverage
+    - add upload step to e2e test for CI RAIWidgets Python Typescript pipeline
+    - add upload step to e2e test to CI notebooks pipeline
+    - return exception details to UI
+    - add docstrings for locale parameter
+  - ## RAIInsights
+    - catch dirty data conditions in RAI Insights validation
+    - update responsibleai to interpret-community 0.24.0
+    - update responsibleai to interpret-community 0.24.1
+    - update responsibleai to interpret-community 0.24.2
+    - make responsibleai managers public and add back compat connectors
+    - make explanations save behavior similar to other managers
+    - change logic to walk categorical columns in validate_train_test_categories
+    - cover missing test scenario in state_directory_management.py
+    - default categorical feature to array
+    - add error analysis test for regression task in RAIInsights
+    - make causal manager add and compute behavior similar to other managers
+  - ## Counterfactual
+    - add feature importance description for counterfactual local importance
+    - update Counterfactual explanation text description
+  - ## Error Analysis
+    - fix importances taking too long to calculate for large data in error analysis
+    - add a perf test for validating training the surrogate error tree on large data and a minor optimization for traversing the tree
+    - add filtering on index to error analysis backend for RAI dashboard cohort switch
+    - remove unnecessary multiple calls to tree api
+    - refactor tree view state to be private static in tree view renderer and not in parent
+    - create github action to release erroranalysis
+    - add excludes cohort filter test in erroranalysis
+    - refactor matrix filter and area state to be private static
+    - make search case insensitive in feature list
+    - add more docs for error analysis surrogate tree
+  - ## Interpret
+    - allow ExplanationDashboard to show limited view for more than 1k features instead of erroring
+  - ## Causal
+    - improve documentation for nuisance_model and heterogeneity_model
+    - implement list method in causal manager
+- ## other
```

Comment:
Would it be good to add explicitly that new pipeline has been created to run RAIwidget e2e tests?

Reply:
I see 'separate out raiwidgets and UI tests into its own workflow' mentioned above. But doesn't mention new pipeline created. So the above question.

Reply:
done!  updated line above to state:
```
separate out raiwidgets and UI tests into its own workflow, and added new pipeline to run RAIwidget e2e tests
```